owners/bootstrap.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+let components = null;
+
+/**
+ * Bootstraps the required components needed either for starting the full Probot
+ * app or just running the info server.
+ *
+ * @param {Logger=} [logger=console] logging interface.
+ * @return {{
+ *   github: !GitHub,
+ *   repo: !Repository,
+ *   ownersBot: !OwnersBot,
+ *   initialized: Promise,
+ * }}
+ */
+function bootstrap(logger) {
```

Comment:
Should this be `bootstrap(logger=console)` in order to set the default value? (I'm not too familiar with setting defaults via JSDoc, and it's not something we commonly do in `amphtml`.)

Reply:
I don't believe so; see [this doc](https://jsdoc.app/tags-param.html#optional-parameters-and-default-values)

Reply:
Gotcha. Relying on documentation for code functionality means our JSDoc must be accurate. For a future general-infrastructure task, we should consider adopting https://www.npmjs.com/package/eslint-plugin-jsdoc for this repo (`amphtml` already uses it to good effect).

Reply:
#568 makes this a moot point.